src/seckrit.py

```diff
@@ -10,11 +10,25 @@
 
 
 def main(args):
+
     # Load the YAML file that tells us which secrets we want to load (and what to do with them).
-    manifest = load_manifest(args.manifest_file)
+    if args.manifest:
+        manifest = load_manifest_from_file(args.manifest)
+    elif "SECKRIT_MANIFEST" in os.environ:
+        manifest = load_manifest_from_string()
+    else:
+        print("ERROR: Should pass --manifest argument or define SECKRIT_MANIFEST environment variable")
+        exit(1)
 
-    # Create the GCP Secret Manager client using the default credential provider chain.
-    client = secretmanager_v1beta1.SecretManagerServiceClient()
+    # Create the GCP Secret Manager client using the credentials stored in the environment otherwise use the default
+    # credential provider chain.
+    credentials_string = os.environ.get("GOOGLE_APPLICATION_CREDENTIALS_JSON")
+    if credentials_string is not None:
```

Comment:
I like the way you used `if "GOOGLE_APPLICATION_CREDENTIALS_JSON" in os.environ` earlier. Perhaps use that format instead, then get the value itself inside the `if` block.